modules/common/src/main/java/com/ibm/cloud/cloudant/security/CouchDbSessionAuthenticator.java

```diff
@@ -108,16 +108,16 @@ public void setHttpConfigOptions(HttpConfigOptions options) {
     /**
      * Setter to configure this authenticator with a new session URL derived from the supplied
      * service URL. This is called internally to apply the service URL from the
-     * { @link com.ibm.cloud.cloudant.v1.Cloudant} client automatically.
+     * <a href="../v1/Cloudant.html"><code>Cloudant</code></a> client automatically.
      *
-     * @param serviceUrl
+     * @param serviceUrl the base service URL associated with the service instance
      */
     public void setSessionUrl(String serviceUrl) {
         this.sessionUrl = RequestBuilder.constructHttpUrl(serviceUrl, new String[]{"_session"});
     }
 
     /**
-     * @param headers
+     * @param headers name value pairs of headers
```

Comment:
This sounds a bit odd.  What about something like:
```suggestion
     * @param headers map of header name and value pairs
```

Reply:
Changed in 639ae25